tenants/ai-example/single-model-serving/base/flan-t5-small/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I would recommend avoiding having multiple bases.

Basically you want to have your overlays inheriting directly from `../../base` but not a folder inside base.

It is ok to use sub folders inside of base to help organize things, but you want a kustomization file in base to inherit the objects in your sub folders.  You don't want your overlays inheriting the subfolders in base.

I am wondering if this might make more sense to just break the flan/granite pieces out into separate examples.

We could deploy flan to all four cluster versions and just do the vllm example for the GPU enabled ones.